pkg/cache/internal/informers_map.go

```diff
@@ -227,6 +233,12 @@ func (ip *specificInformersMap) addInformerToMap(gvk schema.GroupVersionKind, ob
 	ni := cache.NewSharedIndexInformer(lw, obj, resyncPeriod(ip.resync)(), cache.Indexers{
 		cache.NamespaceIndex: cache.MetaNamespaceIndexFunc,
 	})
+
+	// Check to see if there is a transformer for this gvk
+	if err := ni.SetTransform(ip.transformers.Get(gvk)); err != nil {
```

Comment:
Is the informer doing the right thing when `SetTransform` is called with a `nil` argument something we can rely on?

Reply:
If `SetTransform` is called with `nil` then no transform will be applied. That is the intent for this line when no transformer is returned for the gvk by `Get`.